packages/sdk-bundles-api/src/replay/sdk-to-bundle/index.ts

```diff
@@ -1,3 +1,43 @@
+import type { LogLevelDesc } from "loglevel";
+import type { Page } from "puppeteer";
+import { ReplayTimelineEntry } from "./timeline.types";
+
+/** Options for replaying user interaction events */
+export interface ReplayUserInteractionsOptions {
+  page: Page;
+  sessionData: unknown;
+  moveBeforeClick: boolean;
+  acceleratePlayback: boolean;
+  virtualTime: VirtualTimeOptions;
+  maxDurationMs?: number;
+  maxEventCount?: number;
+  onTimelineEvent: OnReplayTimelineEventFn;
+  logLevel: LogLevelDesc;
+}
+
+/** Result of replaying user interactions */
+export type ReplayUserInteractionsResult =
```

Comment:
This type should probably sit in `replay/bundle-to-sdk` folder, and then be imported here. This is because e.g. it's non-breaking to add a new required field to ReplayUserInteractionsResultFull, but breaking to add a new required field to ReplayUserInteractionsOptions; it's non-breaking to delete a field from ReplayUserInteractionsOptions but breaking to delete one from ReplayUserInteractionsResultFull etc. `bundle-to-sdk` would then have a different README.md in it.

(or even better we move ReplayUserInteractionsResult to `replay/bundle-to-sdk`, and OnReplayTimelineEventFn to `sdk-bundles-api/src/functions.ts`, or something and `functions.ts` imports from both bundle-to-sdk and sdk-to-bundle, thereby reflecting the fact that ReplayUserInteractionsFn encodes messages that travel in both directions).